spyder/plugins/completion/languageserver/plugin.py

```diff
@@ -290,17 +280,19 @@ def receive_response(self, response_type, response, language, req_id):
                 self.COMPLETION_CLIENT_NAME, req_id, response)
 
     def send_request(self, language, request, params, req_id):
-        if language in self.clients:
+        if self.enabled and language in self.clients:
+            # Ensure client is started
+            self.start_client(language)
```

Comment:
Some comments here:
1. We should remove all references to self.enabled, as it is True by default
2. On which cases should the client be started from scratch here? We currently enable them if a file is opened, or during startup, maybe there's some corner case we're not taking into account?

Reply:
ah yes this was added when LSP could be enabled in the middle of a session, but that's no longer possible. If you prefer, I can revert these changes as well.

Reply:
I removed all this stuff, since we can always add it back in the future if we need it.